src/Native/System.Native/pal_networking.h

```diff
@@ -117,6 +117,23 @@ struct LingerOption
     int32_t Seconds; // Number of seconds to linger for
 };
 
+struct IOVector
+{
+    uint8_t* Base;
+    uintptr_t Count;
+};
+
+struct MessageHeader
+{
+    uint8_t* SocketAddress;
```

Comment:
For my education - what about when marshalling Ansi strings? Is it OK to use `const char*` then?